ynput/ayon-core
# TVPaint: Move client code to server

## Changelog Description
Moved TVPaint addon logic next to server codebase. This is one more step towards splitting addons to their repos.

## Additional info
From now on TVPaint addon lives out of ayon-core codebase.

## Testing notes:
1. Create tvpaint package (`python ./server_addons/create_ayon_addons.py --addon tvpaint`).
2. Create core package too.
3. Upload both packages to server.
4. Use boths addon in bundle.
5. Integration in TVPaint should work.
